agent-config.yaml

```diff
@@ -0,0 +1,20 @@
+service_name: pythonagent_001
```

Comment:
I am curious about this file, do we need this? I see there is another one for testing so not sure we should include this. Could you elaborate a bit @rcbjBlueMars ?

Reply:
@jcchavezs  we don't need this file and can be removed. I'll remove this file.